### PR TITLE
I18n tooling

### PR DESCRIPTION
**Pull request by ann-kilzer**
I wrote a python script to help automate i18n file management. It uses the [Fire library](https://google.github.io/python-fire/guide/) and is meant to be run as a command line tool:

## Requirements
- [Python 3](https://www.python.org/downloads/) (On Mac OS X, all commands will be `python3` because the OS installed python is the old, obsolete one. If on another system, update all commands `python3` and `pip3` to remove the `3`)
- [pip](https://pypi.org/project/pip/)

```sh
# Use pip to install the project dependencies
pip install -r requirements.txt
```

## Running the tool

```bash
# go into the tools dir
cd tools

# show the man 👨 page for the tool
python3 translation_manager.py 

# Sorts keys in alphabetical order. Useful for consistent diffs
python3 translation_manager.py alphabetize

# copy new keys from en.json to other files
python3 translation_manager.py copyNewKeys

# run the unit tests

python3 test_translation_manager.py
```

Eventually I will wire this tool into GitHub actions, so not everyone needs to run it. I'd like to minimize the number of tools volunteers have to adopt, but I also really love python for quick and dirty data manipulation.

TODO:
- [x] requirements.txt
- [x] write unit tests
- [ ] update README

### CODE REVIEW

File: tools/translation_manager.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import fire
+import json, copy, glob
+
+#from typing import Dict
```
Comment (renderf0x) [inline tools/translation_manager.py:6]: Nit: Extraneous commented-out import?

File: tools/translation_manager.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      for fname in get_locale_files():
+          print(f"Updating {fname}...")
+          data = openLocaleFile(fname)
+          export(data, fname)
+      print("Done!🙌🏻")
+          
+  
+  def copy_new_keys(self, locale="en"):
+    """Moves all missing keys from the primary locale (default is English) to the other locale files"""
+    primaryFName = f"../locales/{locale}.json"
```
Comment (renderf0x) [inline tools/translation_manager.py:63]: Nit: Path could be a constant perhaps? Using it in a couple of places.

File: tools/test_translation_manager.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import unittest
+
+from translation_manager import *
+
+class TestTranslationManager(unittest.TestCase):
+
+    def test_copy_new_keys_to_locale(self):
+        self.assertEqual('foo'.upper(), 'FOO')
```
Comment (renderf0x) [inline tools/test_translation_manager.py:10]: Nit: Is this leftover from making the tests?

**Review by renderf0x — APPROVED** [review]
LGTM. Couple of nits but no issues.

File: tools/test_translation_manager.py (hunk 3 above)
Comment (ann-kilzer) [reply tools/test_translation_manager.py]: yes i missed those

**Pull request merged**